torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -1,13 +1,15 @@
 from collections import defaultdict
 import logging
 import math
+import copy
```

Comment:
nit: sort imports (there should be a menu option in VS Code)

Reply:
Thanks for pointing out this option in VS Code. Sorted inputs in the updated PR.